src/app/modules/form/public-form/public-form.controller.ts

```diff
@@ -59,15 +59,7 @@ const validateSubmitFormFeedbackParams = celebrate({
 })
 
 /**
- * NOTE: This is exported solely for unit testing
- * Handler for POST /:formId/feedback endpoint
- * @precondition formId should be present in req.params.
- * @precondition Joi validation should enforce shape of req.body before this handler is invoked.
- *
- * @returns 200 if feedback was successfully saved
- * @returns 404 if form with formId does not exist or is private
- * @returns 410 if form has been archived
- * @returns 500 if database error occurs
+ * @deprecated use submitFormFeedbackV2 instead
```

Comment:
we can add a TODO comment here linking to the cleanup issue you created, so we can just search for the issue number and see all the places where we need to clean up